src/Nox.Solution/Models/Types/TypeUserInterface/TypeUserInterface.cs

```diff
@@ -2,22 +2,59 @@
 
 namespace Nox.Solution
 {
-
+    [Title("UI display options for this attribute.")]
+    [Description("Specify how this attribute is rendered on the Nox user interface. Configuration options include label, icon, input/output masks, help hints among other.")]
```

Comment:
On any interface... you can remove "Nox" - someone may build better interfaces using these definitions

Reply:
Done, have removed UI acronyms in favour of full text.